prime-router/build.gradle.kts

```diff
@@ -122,6 +122,7 @@ compileKotlin.kotlinOptions.allWarningsAsErrors = true
 compileKotlin.kotlinOptions.useK2 = false
 compileTestKotlin.kotlinOptions.jvmTarget = "11"
 compileTestKotlin.kotlinOptions.allWarningsAsErrors = true
+compileTestKotlin.kotlinOptions.allWarningsAsErrors = true
```

Comment:
Redundant?

Reply:
Remove

Reply:
@carlosfelix2 @snesm this is for the tests and not for the main body of code. Do you still want me to remove it?